components/contribution-flow/index.js

```diff
@@ -171,12 +172,21 @@ class ContributionFlow extends React.Component {
         variables: {
           order: {
             quantity: stepDetails.quantity,
-            amount: { valueInCents: stepDetails.amount },
+            amount:
+              this.props.paymentMethod === 'crypto'
+                ? { value: stepDetails.amount }
```

Comment:
Doesn't not make sense, right? The value should be dummy if I'm not mistaken.

Reply:
Yes, you are right, I've corrected it to always put the value 100 cents (1$). 👍🏽